<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NarcoCANtrol</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="receivedMessagesBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>960</width>
      <height>620</height>
     </rect>
    </property>
    <property name="title">
     <string>Log Window</string>
    </property>
    <widget class="LogWindow" name="logWindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>940</width>
       <height>580</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
    <widget class="QLabel" name="busStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>940</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="messageFilterSettingsBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>190</width>
      <height>620</height>
     </rect>
    </property>
    <property name="title">
     <string>Message Filter Settings</string>
    </property>
    <widget class="QGroupBox" name="filterFCodesBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>170</width>
       <height>250</height>
      </rect>
     </property>
     <property name="title">
      <string>F-Codes</string>
     </property>
     <widget class="QCheckBox" name="filterReadRegsRange">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>95</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Regs Range</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterReadRegsSeries">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Regs Series</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterDeviceSpecific_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>145</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device-specific (1)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterAllFCodes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>All F-Codes</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterWriteRegsRange">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>45</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Write Regs Range</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterWriteRegsSeries">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Write Regs Series</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterDeviceSpecific_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device-specific (2)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterDeviceSpecific_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>195</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device-specific (3)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterDeviceSpecific_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device-specific (4)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="filterMsgTypesBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>170</width>
       <height>150</height>
      </rect>
     </property>
     <property name="title">
      <string>Message Types</string>
     </property>
     <widget class="QCheckBox" name="filterMaster">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>95</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Master's Request</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterSlave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Slave's Response</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterAllMsgTypes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>All Message Types</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterHighPrioMaster">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>45</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Master's High-prio</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterHighPrioSlave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Slave's High-prio</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionClearLog"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuCalls"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionClearLog"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the CAN Device</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/disconnect.png</normaloff>:/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from the CAN Device</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../images/images.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWindow</class>
   <extends>QTableWidget</extends>
   <header>src/main/log_window.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
